examples/simple-platform/user-walk-through.md

```diff
@@ -0,0 +1,48 @@
+# User Walk-through of the Platform Example
+
+## Creating an account and going through the GitHub OAuth loop
+
+Open the accounts app in your browser at `http://localhost:3025`. If you open 
+your browser's developer tools to the Console tab, you can see some 
+`interbit-middleware` commands being printed. The accounts app is fully up and 
+running once you see the `interbit-middleware/CHAIN_BLOCKING` status for the 
+`accountsPrivate` chain. This may take up to a minute. 
+
```

Comment:
It really shouldn't take up to a minute. It does take longer the first time.

It is also useful to clarify what you mean by fully up and running. What you mean is that you can dispatch actions to your private account chain.